recipe/recipe.yaml

```
@@ -0,0 +1,85 @@
+context:
+  name: pyright
+  version: 1.1.386
+
+recipe:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+build:
+  number: 10
+
+source:
+  url: https://pypi.io/packages/source/${{ name[0] }}/${{ name }}/pyright-${{ version }}.tar.gz
+  sha256: 8e9975e34948ba5f8e07792a9c9d2bdceb2c6c0b61742b068d2229ca2bc4a9d9
+
+outputs:
+  - package:
+      name: ${{ name|lower }}-node
+    build:
+      skip: [ppc64le]
+      script:
+        file: build_node${{ '.bat' if win else '.sh' }}
+    requirements:
+      build:
+        - pnpm
+        - pnpm-licenses
+      host:
+        - nodejs
+    about:
+      license: MIT
+      license_file: third-party-licenses.txt
+
+  - package:
+      name: ${{ name|lower }}
+    build:
+      noarch: python
```

```suggestion
      noarch: python
      skip: not linux-64
```
something like this since this is noarch? (not 100% sure about the syntax)